website/content/concepts/layer2.md

```diff
@@ -77,10 +78,10 @@ Keepalived uses the Virtual Router Redundancy Protocol (VRRP). Instances of
 Keepalived continuously exchange VRRP messages with each other, both to select a
 leader and to notice when that leader goes away.
 
-MetalLB on the other hand relies on Kubernetes to know when pods and nodes go up
-and down. It doesn't need to speak a separate protocol to select leaders,
-instead it just lets Kubernetes do most of the work of deciding which pods are
-healthy, and which nodes are ready.
+MetalLB on the other hand relies on
+[memberlist](https://github.com/hashicorp/memberlist) to know when a Node in
+the cluster is no longer reachable and the service IPs from that node should be
+moved elsewhere.
```

Comment:
We still use the K8S API for the pods status, but not sure how to convey that

Reply:
Sure, how about this.  I'm not even sure if this is right, I'm guessing.  :-)

> MetalLB on the other hand relies on [memberlist](https://github.com/hashicorp/memberlist) to know when a Node in the cluster is no longer reachable or the Kubernetes API to determine if a MetalLB pod (speaker) is unhealthy. This information is used to determine if service IPs from that node should be moved elsewhere.

Reply:
I was thinking of the pods that host the services, not the speaker pod.

Reply:
Oh, I see what you meant.

Since this text mainly seemed about the failover part, I'm not sure how to mention that.  I think it's ok how it is.

Reply:
... and now that I've actually read the code to see how it works!  How about:

```
MetalLB on the other hand relies on
[memberlist](https://github.com/hashicorp/memberlist) to know when a Node in
the cluster is no longer reachable and the service IPs from that node should be
moved to another Node that hosts an endpoint for each affected Service.
```

Reply:
@champtar IMO this is fine. But not sure I understood what your concern was? :)

Reply:
@russellb last proposal is good

Reply:
Cool, marking as solved, then! Thanks! :)
(let me know if I'm misunderstanding)